src/components/navbar.tsx

```diff
@@ -1,18 +1,64 @@
+"use client";
 import styled from "@emotion/styled";
 import Link from "next/link";
 import { usePathname } from "next/navigation";
+import chatIcon from "../../public/svg/icon-chat.svg";
+import burgerIcon from "../../public/svg/icon-burger.svg";
+import closeIcon from "../../public/svg/icon-close.svg";
+import Image from "next/image";
+import { useEffect, useState } from "react";
 
 const StyledNav = styled.nav`
   display: flex;
   position: fixed;
-  bottom: 8px;
-  left: 25%;
-  right: 25%;
-  margin: auto;
-  border-radius: 1rem;
+  padding: 0 0.5rem;
+  border-radius: 0.75rem;
+  background-color: rgba(57, 151, 156, 0.2);
+  width: max-content;
+  backdrop-filter: blur(5px);
+
+  @media (min-width: 700px) {
+    height: "auto";
+    bottom: 16px;
+    left: 5%;
+    right: 5%;
+    margin: auto;
+
+    @media (max-width: 700px) {
+      display: none;
+    }
+  }
+`;
+
+const StyledNavMobile = styled.nav<{ isMenuOpen: boolean }>`
```

Comment:
<img width="134" alt="Screenshot 2024-02-16 at 15 01 42" src="https://github.com/web-space-dev/website-frontend/assets/29145846/6cf79955-a96d-42ef-af23-4fe6b37887cb">

The X icon is stretched, could we try have this more even ( maybe for now you could even make this dropdown full width - until we get the designs)

Reply:
![Screenshot 2024-02-17 at 12 59 48](https://github.com/web-space-dev/website-frontend/assets/147092788/f93c9582-9fd1-4ce1-b3ea-bb7400e143c9)

In addition, I disabled  the background Scroll when the menu is open.